renku/service/entrypoint.py

```diff
@@ -62,6 +66,13 @@
 
 logging.basicConfig(level=os.getenv("SERVICE_LOG_LEVEL", "WARNING"))
 
+if os.getenv("SENTRY_DSN"):
```

Comment:
Since both of them are used, should we check for both of them here?

```suggestion
if os.getenv("SENTRY_DSN") and os.getenv("SENTRY_ENV"):
```

Reply:
not sure - the environment doesn't need to be set for sentry to work, it depends a bit on how sentry is used. The DSN otoh is required to activate sentry.